apps/prairielearn/src/lib/config.ts

```diff
@@ -150,7 +150,6 @@ const ConfigSchema = z.object({
   githubCourseOwner: z.string().default('PrairieLearn'),
   githubCourseTemplate: z.string().default('pl-template'),
   githubMachineTeam: z.string().default('machine'),
-  githubMainBranch: z.string().default('master'),
```

Comment:
AFAICT making this configurable is unnecessary. We always assert that a new repo has exactly one branch, and that branch must always have a name, so we should never hit a case where we need to fall back to this value.